include/deal.II/base/mpi_remote_point_evaluation.h

```diff
@@ -23,6 +23,8 @@
 
 #include <deal.II/dofs/dof_handler.h>
 
+#include <deal.II/grid/grid_tools.h>
```

Comment:
Could we forward declare `DistributedComputePointLocationsInternal`?

Reply:
Yes. Done in [bbeaaeb](https://github.com/dealii/dealii/pull/15154/commits/bbeaaebe2d7af6039d0457ece172cd73871f16b1)